### ISSUE TITLE
issue with v0.3.11 causing, infinite loop error

### ISSUE THREAD

**Issue by arunoda** [arunoda/meteor-smart-collections#30]
With this v0.3.11 released, there is an infinite error. Which gives this error message.

```
events.js:72
        throw er; // Unhandled 'error' event
              ^
RangeError: Maximum call stack size exceeded
```

Possible fix is on branch: `fix-crash-issue`

**Comment by arunoda** [comment]
Revert back to original oplog logic: 9ab9d92 
Waiting for the approval

**Comment by arunoda** [comment]
Confirmed this fixes the issue

**Comment by arunoda** [comment]
published with the `0.3.13`
